test/SimpleLINQ.Test/BasicUsage.cs

```diff
@@ -182,7 +182,7 @@ public void CanCallToList()
         public void CanCallAny()
         {
             IQueryable<Foo> query = CreateQuery();
-            query.Where(x => x.Bar == "abc" && x.Blap == 123).Any();
+            query.Any(x => x.Bar == "abc" && x.Blap == 123);
```

Comment:
If anything, this should be an extra test, i.e. check both ways; they are different code paths